site/nav.yml

```diff
@@ -23,23 +23,23 @@ nav:
   - Docs: 
     - nightly: '!include docs/docs/nightly/mkdocs.yml'
     - latest: '!include docs/docs/latest/mkdocs.yml'
```

Comment:
We still need 1.9.1 since there's no version info on this "latest" doc.

Reply:
Good point. Added d8e56db604617b8ee253be65f98c27d59b3d5d12